backend/app/services/web_scraper/profiles.py

```
@@ -0,0 +1,61 @@
+                "AppleWebKit/605.1.15 (KHTML, like Gecko) Version/17.0 "
+                "Mobile/15E148 Safari/604.1"
+            ),
+            user_agent_mode=None,
+            viewport={"width": 390, "height": 844},
+        ),
+        "managed_session": BrowserProfile(
+            name="managed_session",
+            user_agent_mode="random",
+            use_managed_browser=True,
+        ),
+    }
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)